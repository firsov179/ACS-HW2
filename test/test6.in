34ehy83h89fr2f83yhvc
