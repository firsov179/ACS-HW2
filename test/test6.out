The minimal sign is "3".
The maximal sign is "y".
